tests/unit/test_config.py

```diff
@@ -73,7 +73,7 @@ def test_build_config_without_s3_data(self):
 
     def test_build_s3_storage_config(self):
         raw_s3_config_data = {
-            "endpoint": "s3.com:port",
+            "endpoint": "http://s3.com:port",
```

Comment:
Are we guaranteed to have a scheme in there? Should the stripping code take caution?

Reply:
The stripping function (before and after your suggestion to use `re.sub()`) returns the base URL regardless; I'll add a separate test for that, just to be safe.